NEWS.md

```diff
@@ -7,6 +7,12 @@
 - New functions `set_thumbnail()`, `get_thumbnail()`, `delete_thumbnail()` and
   `has_thumbnail()` let you interact with content thumbnails, replacing older
   `*_image()` functions. (#294, #304)
+- `groups_create_remote()` now has an `exact` parameter. Setting `exact` causes
+  the function to consider only exact group name matches when searching for
+  remote groups and checking for existing local groups. (#216)
+- Previously, `groups_create_remote()` and `users_create_remote()` could return
+  a user or group other than the group(s) created. Now they always return the
+  details of the newly-created user(s) or group(s) in a list.
```

Comment:
`groups_create_remote()` was changed back and should be removed from the entry
```suggestion
- Previously, `users_create_remote()` could return a user or group other than the
  group(s) created. Now they always return the details of the newly-created user(s)
  or group(s) in a list.
```

If `users_create_remote()` is also changed, this bullet should be removed.
```suggestion
```

Reply:
Accepting the suggestion locally  :)